### PR TITLE
Add Owntracks and Calagator

### PR DESCRIPTION
**Pull request by ligurio**
Thank you for taking the time to work on a PR for Awesome-selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -595,6 +595,7 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
   * [MapBBCodeShare](https://github.com/MapBBCode/share.mapbbcode.org) - Tool for sharing custom OSM maps. Support for annotated markers, polygons, lines, multi-format import/export, multiple layers, shortlinks... ([Demo](http://share.mapbbcode.org/)) `WTFPL/Other` `PHP/Javascript`
   * [OpenGTS](http://www.opengts.org/) - Entry-level fleet tracking system. Supports variety of tracking devices and protocols. Comes with rich web-interface and reporting features. ([Demo](http://track.opengts.org/track/Track), [Source Code](https://sourceforge.net/projects/opengts/files/server-base/)) `Apache` `Java`
   * [Openstreetmap](http://www.openstreetmap.org/) - OpenStreetMap is a map of the world, created by people like you and free to use under an open license. ([Source Code](https://github.com/openstreetmap/openstreetmap-website))
+  * [Owntracks](http://owntracks.org/) - Self-hosted, opensource location tracking. ([Source code](https://github.com/owntracks)) `GPLv3`
```
Comment (n8225) [inline README.md]: Missing language

File: README.md [hunk 2]
```diff
@@ -766,6 +767,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 ## Polls and Events
 
+  * Calagator - An event aggregator. ([Demo](http://calagator.org/), [Source code](https://github.com/calagator/calagator)) `MIT` `Ruby`
```
Comment (n8225) [inline README.md]: Please follow the guidelines in [Contributing](https://github.com/Kickball/awesome-selfhosted/blob/master/.github/CONTRIBUTING.md) for example:
> \[Calagator\]\(http://calagator.org/)

**Comment by n8225** [comment]
Merge conflicts must corrected as well.

**Comment by Kickball** [comment]
@ligurio have you had chance to work on this?

**Comment by AndyR207** [comment]
@ligurio Please work on the changes requested to this PR or it will be closed in 3 days.

**Comment by ligurio** [comment]
@AndyR207 feel free to close, I have no free time for this

**Pull request closed without merging**